spec/spec_helper.rb

```diff
@@ -15,6 +15,7 @@
 RSpec.configure do |config|
   config.infer_base_class_for_anonymous_controllers = true
   config.use_transactional_fixtures = false
+  config.filter_run_excluding type: :controller unless Object.const_defined?(:APIApplicationController)
```

Comment:
hm, yes that'd be good, then it'll match the same test I used in `lib/activerecord-multi-tenant.rb`.

Just pushed an update to do so.